react-example/public/index.html

```diff
@@ -0,0 +1 @@
+<div id="root"></div>
```

Comment:
should we have at least some HTML boilerplate here?

Reply:
We don't need as react-scripts auto generates that